src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -303,6 +305,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			if (contents) {
 				this._defaultLanguageInfo = contents.metadata && contents.metadata.language_info;
 				this._savedKernelInfo = this.getSavedKernelInfo(contents);
+				if (contents.metadata) {
```

Comment:
When saving a notebook, won't the metadata be lost? See https://github.com/microsoft/azuredatastudio/pull/9914 :)

Reply:
merged the metadata changes

Reply:
Please don't merge in unrelated changes when doing a PR - it makes it difficult to review. (You can keep the changes locally for testing)

Reply:
This was merge conflict. The changes are already in master

Reply:
You should merge in latest from master then so these changes don't show up in this PR.